frontends/web/src/components/anchor/anchor.tsx

```diff
@@ -14,36 +15,49 @@
  * limitations under the License.
  */
 
-import { ReactNode, SyntheticEvent, useContext } from 'react';
-import { route } from '../../utils/route';
-import { debug } from '../../utils/env';
+import { ReactNode, SyntheticEvent } from 'react';
 import { apiPost } from '../../utils/request';
-import AppContext from '../../contexts/AppContext';
 import style from './anchor.module.css';
 
 type TProps = {
-    href: string;
-    children: ReactNode;
-    [property: string]: any;
```

Comment:
`property: string]: any;` did basically allow everything before (also `onClick={() => {}}`) .

Reply:
Why would we have this before 😄  pretty random.